scene/gui/slider.cpp

```diff
@@ -270,13 +278,16 @@ void Slider::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("set_editable", "editable"), &Slider::set_editable);
 	ClassDB::bind_method(D_METHOD("is_editable"), &Slider::is_editable);
+	ClassDB::bind_method(D_METHOD("set_grabber_offset", "grabber_offset"), &Slider::set_grabber_offset);
+	ClassDB::bind_method(D_METHOD("get_grabber_offset"), &Slider::get_grabber_offset);
 	ClassDB::bind_method(D_METHOD("set_scrollable", "scrollable"), &Slider::set_scrollable);
 	ClassDB::bind_method(D_METHOD("is_scrollable"), &Slider::is_scrollable);
 
 	ADD_SIGNAL(MethodInfo("drag_started"));
 	ADD_SIGNAL(MethodInfo("drag_ended", PropertyInfo(Variant::BOOL, "value_changed")));
 
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "editable"), "set_editable", "is_editable");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "grabber_offset"), "set_grabber_offset", "get_grabber_offset");
```

Comment:
This could make sense as a theme property, because you want a single value for the given grabber texture.